src/main/java/org/jenkinsci/test/acceptance/docker/fixtures/SvnContainer.java

```diff
@@ -45,7 +44,7 @@ public URL getHttpUrl() throws SubversionPluginTestException {
      * @throws SubversionPluginTestException e
      */
     public URI getSvnUrl() throws SubversionPluginTestException {
-        String url = PROTOCOL_SVN + LOCALHOST + port(3690);
+        String url = PROTOCOL_SVN + ipBound(80) + ':' + port(3690);
```

Comment:
it could do - but both the ssh daemon and the web interface will be on the same machine so....